faiss/cppcontrib/sa_decode/Level2-inl.h

```diff
@@ -12,6 +12,15 @@
 #include <cstdint>
 
 #include <faiss/cppcontrib/detail/CoarseBitType.h>
+#include <faiss/impl/platform_macros.h>
+
+bool isBigEndian() {
```

Comment:
please do not put this in the global namespace. At least put it in faiss::cppcontrib.

Reply:
Okay. I will change this.

Reply:
@mdouze please check now